### PR TITLE
specs: Move to dedicated repo

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

**Review by protolambda — COMMENTED** [review]
`specs/meta/devnet.md`, `specs/meta/linting.md`, `specs/meta/versioning.md` may find a better new home in the `docs` dir. I'm fine with removing these though, as the `CONTRIBUTING.md` already covers these topics, and automation is better than the manual commands in these old meta docs.

Also, maybe we can remove the `specs` link from the README dir tree-view, and add a new separate section that links to the specs repository?

File: specs/README.md [hunk 1]
```diff
@@ -1,63 +1,3 @@
-<!-- DOCTOC SKIP -->
-# Optimism Bedrock specs
+# Specs Have Moved
```
Comment (axelKingsley) [inline specs/README.md:1]: At some point Specs will have lived in the new location for long enough that we won't consider them "moved". At that point we could remove the whole `specs` folder. Considering you've already changed the specs pointer in the main README, we could remove this folder any time IMO.

**Review by axelKingsley — APPROVED** [review]
Looks clean!

Lines which are "added" are modifications to disconnect linting or otherwise point at the new location.

File: specs/README.md (hunk 1 above)
Comment (mslipper) [reply specs/README.md:1]: Will remove in a month or so I think - there could be links on the web to stuff in that folder, so if people click around I want to make it clear where to go. GitHub doesn't support redirects in repositories, sadly.

**Review by trianglesphere — APPROVED** [review]

**Review by refcell — APPROVED** [review]
LGTM

**Pull request merged**